### DISCUSSION TITLE
Online Documentation

### DISCUSSION THREAD

**Discussion by smchargue** [pnp/powershell#262] [General] [answered]
Where did the great online documentation go?  It was here, 
https://docs.microsoft.com/en-us/powershell/module/sharepoint-pnp/?view=sharepoint-ps 
now it's not.

**Comment by xlawsdm** [comment]
I confirm it is all gone. What is going on?

**Comment by erwinvanhunen** [comment, marked as answer]
We're in the progress of updating it, unfortunately the process is a bit slow. You can find the current documentation -also- at

https://pnp.github.io/powershell

In the top navigation click 'Cmdlets'.

**Reply by erwinvanhunen** [reply]
Also keep in mind that -all- cmdlet documentation is avaiable -inside- PnP PowerShell

Get-Help [cmdletname]

**Comment by b-dasass** [comment]
Will you keep the "legacy" documentation available? If you plan to retire that branch of the documentation, could you please archive it together with the legacy code base? Thank you!

**Reply by erwinvanhunen** [reply]
The legacy documentation can be (I realize it's not -that- user friendly to browse it that way) here: https://github.com/MicrosoftDocs/office-docs-powershell/tree/74c4a9b5153977e1db1fe0d1bc539839806f09d0/sharepoint/sharepoint-ps/sharepoint-pnp
